# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux26/95-adb-mapping.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Well, after fixing user-space apps here and there it became too annoying
and counter productive, since nowadays user-space expect full "i386 style",
linux mapping anyways. So let's correct the alt and altgr mapping in the
kernel in the first place.

  - Rene Rebe <rene@exactcode.de>

--- ./drivers/macintosh/adbhid.c.orig	2004-10-18 23:55:36.000000000 +0200
+++ ./drivers/macintosh/adbhid.c	2004-10-19 18:58:36.972070126 +0200
@@ -130,10 +130,10 @@
 	/* 0x34 */ KEY_KPENTER,		/*  96 */
 	/* 0x35 */ KEY_ESC,		/*   1 */
 	/* 0x36 */ KEY_LEFTCTRL,	/*  29 */
-	/* 0x37 */ KEY_LEFTMETA,	/* 125 */
+	/* 0x37 */ KEY_LEFTALT,		/* 125 */
 	/* 0x38 */ KEY_LEFTSHIFT,	/*  42 */
 	/* 0x39 */ KEY_CAPSLOCK,	/*  58 */
-	/* 0x3a */ KEY_LEFTALT,		/*  56 */
+	/* 0x3a */ KEY_RIGHTALT,	/*  56 */
 	/* 0x3b */ KEY_LEFT,		/* 105 */
 	/* 0x3c */ KEY_RIGHT,		/* 106 */
 	/* 0x3d */ KEY_DOWN,		/* 108 */
